pact/marmalade.repl

```diff
@@ -1,2 +1,348 @@
-(load "./policy-manager/policy-manager.repl")
-(typecheck "marmalade-v2.ledger")
+(begin-tx)
+  (env-data
+    { 'ns-admin-keyset: []
+    , 'ns-genesis-keyset:[]
+    , 'ns-operate-keyset:[] })
+  (load "./root/fungible-v2.pact")
+  (load "./root/fungible-xchain-v1.pact")
+  (load "./root/gas-payer-v1.pact")
+  (env-exec-config ["DisablePact44"])
+  (load "./root/ns.pact")
+  (define-namespace 'kip (sig-keyset) (sig-keyset))
+
+  (load "./kip/account-protocols-v1.pact")
+  (env-data
+  { 'marmalade-admin: ["marmalade-admin"]
+  , 'marmalade-ns-user: ["marmalade-admin"]
+  , 'marmalade-ns-admin: ["marmalade-admin"]
+  , 'ns: "marmalade-v2"
+  , 'upgrade: false })
+
+  (load "./kip/manifest.pact")
+  (load "./kip/token-policy-v2.pact")
+  (load "./kip/poly-fungible-v3.pact")
+  (define-namespace 'util (sig-keyset) (sig-keyset))
+  (load "./util/fungible-util.pact")
+  (load "./util/guards1.pact")
+
+(commit-tx)
+
+;; todo: move this to a util lib
```

Comment:
Can we take care of this todo right away? I agree with moving it to a util lib, but these todo's tend to stick around if we don't tackle them immediately

Reply:
Agree with the point. i've moved to `marmalade-v2.util-v1` contract, open to discussion about its placement